src/plugins/candyMachineModule/CandyMachineClient.ts

```diff
@@ -16,7 +17,7 @@ export type CandyMachineInitFromConfigOpts = {
 };
 
 export class CandyMachineClient extends ModuleClient {
-  findCandyMachineByAddress(address: PublicKey): Promise<CandyMachine> {
+  findCandyMachineByAddress(address: PublicKey): Promise<CandyMachine | null> {
```

Comment:
The NFT module throws "not found" errors on its `findX` methods. Do we think we should return `null` instead? It's a bit problematic when composing multiple handlers together.

Reply:
Well the candy machine might not exist, so not sure if we should throw here. Users can check for `null`.

But let's agree on a common pattern and adjust accordingly.

Reply:
Let's keep it as-is for now and talk about it later to sync the modules together.